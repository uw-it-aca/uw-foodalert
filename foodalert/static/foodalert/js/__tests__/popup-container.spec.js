"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import PopupContainer from '../components/popup-container.vue';

describe('Labelled Input', function () {
    describe('renders correctly', function () {
        test('textarea', function () {
            var wrapper = mount(PopupContainer, {
                propsData: {
                }
            });
            expect(wrapper.element).toMatchSnapshot();
        });
    });
})
