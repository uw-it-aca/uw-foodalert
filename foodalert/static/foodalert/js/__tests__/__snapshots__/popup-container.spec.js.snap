// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Labelled Input renders correctly textarea 1`] = `
<div>
  <div
    id="modal-null___BV_modal_outer_"
  >
    <div
      aria-hidden="true"
      class="modal fade"
      id="modal-null"
      role="dialog"
      style="display: none;"
    >
      <div
        class="modal-dialog modal-md"
      >
        <div
          aria-describedby="modal-null___BV_modal_body_"
          class="modal-content"
          role="document"
          tabindex="-1"
        >
          <!---->
          <div
            class="modal-body"
            id="modal-null___BV_modal_body_"
          />
          <!---->
        </div>
      </div>
    </div>
    <!---->
  </div>
</div>
`;
